docs/Esport Protocol V1/Tech integration/tournaments.md

```diff
@@ -0,0 +1,322 @@
+---
+sidebar_position: 3
+---
+
+# Tournaments Endpoints
+
+The Tournaments Endpoints provides a comprehensive suite of functions for managing and participating in esports tournaments. This API allows players to join or leave tournaments, update scores, retrieve tournament details, and access leaderboards.
+
+## API Documentation: Join Tournament API
+
+### Endpoint: POST `/fe/esports/tournaments/{tournamentId}/join`
+This endpoint is used for registering a player to participate in a specific tournament, capturing their entry into the competitive event.
+
+##### Request Header
+- `Content-Type`: application/json
+- `user-jwt-token`: Users JWT token for authentication.
+
+#### Request Structure
+
+##### `joiningStake`
+   - **Type:** int
+   - **Description:** Entry fee of the tournament.
+
+#### Response
+- **`response`:** 
+  - **Type:** Object
+  - **Description:** Indicates if the operation was successful.
+  - **Field:** `"message"`
+  - **Value:** `"user successfully joined the tournament"`
+
+##### Response Status Code
+- `Success`: 200 OK
+- `Failure`: 
+  - `400 Bad Request`
+    - "TOURNAMENT_INVALID_JOINING_STAKE"
+    - "TOURNAMENT_IS_NOT_LIVE"
+  - `404 Not Found`
+    - "TOURNAMENT_NOT_FOUND"
+  - `409 Conflict`
+    - "EXISTING_PARTICIPATION_FOUND"
+    - "TOURNAMENT_IS_FULL"
+
+##### Example Response
+```json
+{
+  "message": "user successfully joined the tournament"
+}
+```
+
+## API Documentation: Leave Tournament API
```

Comment:
Can we remove **API Documentation** everywhere, Let's keep the purpose alone like 
Join Tournament API, Leave Tournament API @Jitendra-08